recipe/meta.yaml

```
@@ -21,26 +21,32 @@ requirements:
     - hatchling
   run:
     - python
-    - deprecated >=1.2.6
-    - opentelemetry-api >=1.3,<2.0.0.dev0
+    - opentelemetry-api >=1.3,<2.dev0
```

https://github.com/open-telemetry/opentelemetry-python/blob/v1.38.0/propagator/opentelemetry-propagator-b3/pyproject.toml#L30
```suggestion
    - opentelemetry-api >=1.3,<2
```
Please don`t use dev builds